### PR TITLE
add a blog post for Fedora ELN

### PR DESCRIPTION
**Pull request by TomasTomecek**
Ready for some early feedback, still wanna tinker some details.

### CODE REVIEW

File: content/posts/fedora-eln.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+But on the other hand, it's kinda too late when the build happens after an
+upstream release is out and being updated in Fedora Rawhide dist-git. Wouldn't
+it be better to know if the upstream checkout builds fine in ELN while working on
+the code?
+
+Oh, wait!
+
+**TL;DR you can do this easily with packit!!**
+
+### The long read
```
Comment (jpopelka) [inline content/posts/fedora-eln.md:23]:
```suggestion
### You can do this easily with packit!
```

**Review by jpopelka — COMMENTED** [review]
LGTM

File: content/posts/fedora-eln.md (hunk 1 above)
Comment (TomasTomecek) [reply content/posts/fedora-eln.md:23]: shoter & better, thanks!

File: content/posts/fedora-eln.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+1. the standard Fedora Rawhide buildroot
+2. while the second one mimics Red Hat Enterprise Linux environment
+
+This way you can make sure that your new upstream release builds fine in the next RHEL.
+
+But on the other hand, it's kinda too late when the build happens after an
+upstream release is out and being updated in Fedora Rawhide dist-git. Wouldn't
+it be better to know if the upstream checkout builds fine in ELN while working on
+the code?
```
Comment (csomh) [inline content/posts/fedora-eln.md:17]:
```suggestion
On the other hand, getting this feedback is kinda too late: the upstream
release already happened and the code was imported in Fedora dist-git,
so fixing an issue will require repeating the whole process. Wouldn't
it be better to know if the upstream change builds fine in ELN **while**
working on the code?
```

File: content/posts/fedora-eln.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+Oh, wait!
+
+**TL;DR you can do this easily with packit!!**
```
Comment (csomh) [inline content/posts/fedora-eln.md:21]:
```suggestion
**TL;DR you can do this easily with Packit!**
```

File: content/posts/fedora-eln.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+      targets:
+        - fedora-development
+        - fedora-eln
+      branch: master
+```
+
+### Why?
+
+If you got here and you're still not sure why you'd need this, I can give
+you a practical example.
```
Comment (csomh) [inline content/posts/fedora-eln.md:63]:
```suggestion
you a real life example.
```

File: content/posts/fedora-eln.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  - job: copr_build
+    trigger: commit
+    metadata:
+      targets:
+        - fedora-development
+        - fedora-eln
+      branch: master
+```
+
+### Why?
```
Comment (csomh) [inline content/posts/fedora-eln.md:60]:
```suggestion
### A real life example
```

File: content/posts/fedora-eln.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+Jirka followed and added Fedora ELN as another target. The build failed on the
+fact that one of build requirements was not available in ELN.
+
+```
+No match for argument: metacity
+```
+
+Since the team discovered this during their upstream development process, they
+can react to the issue right away. It would be pretty late if they found this
```
Comment (csomh) [inline content/posts/fedora-eln.md:80]:
```suggestion
can react to the issue right away. It would have been pretty late if they found this
```

File: content/posts/fedora-eln.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+out while the next major RHEL is reaching alpha - at this moment they should
```
Comment (csomh) [inline content/posts/fedora-eln.md:81]:
```suggestion
while the next major RHEL is reaching alpha - at this moment they should
```

File: content/posts/fedora-eln.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+title: "Working on the next major RHEL release, in your upstream repo"
+date: 2020-10-04T12:07:03+02:00
+draft: false
+---
+
+[Fedora EL Niño](https://docs.fedoraproject.org/en-US/eln/) (ELN) is such an
+awesome idea. It enables building rawhide packages in two distinct buildroots:
+
+1. the standard Fedora Rawhide buildroot
+2. and the second one, which mimics Red Hat Enterprise Linux
```
Comment (csomh) [inline content/posts/fedora-eln.md:11]:
```suggestion
1. the standard Fedora Rawhide buildroot and
2. a second one, which mimics Red Hat Enterprise Linux
```

File: content/posts/fedora-eln.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+
+This way you can make sure that your new upstream release builds fine in the
+next RHEL.
+
+It's a little unfortunate we are getting this feedback too late: the upstream
```
Comment (csomh) [inline content/posts/fedora-eln.md:16]:
```suggestion
But this feedback might be a little bit too late: the upstream
```

My main concern here was that the previous sentence is using "you", this one used "we", and felt confusing. Made it impersonal to fix this.

File: content/posts/fedora-eln.md [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+```
+
+With this config, changes from every pull request will be built in all
+development versions of Fedora (at the time of writing this, it's Rawhide and
+Fedora 33) and in Fedora ELN.
+
+Easy, right?
+
+You can set up the same thing for pushes to a branch, for example build the
+master branch in ELN and development versions of Fedora:
```
Comment (csomh) [inline content/posts/fedora-eln.md:49]:
```suggestion
You can set up the same to happen when pushing to a branch, for example build the
`master` branch in ELN and development versions of Fedora:
```

**Review by csomh — COMMENTED** [review]
A few more recommendations to improve clarity and simplify the language.

File: content/posts/fedora-eln.md (hunk 10 above)
Comment (TomasTomecek) [reply content/posts/fedora-eln.md:49]: I disagree on this one - I prefer my wording since it's shorter and I feel like the word "happen" is implied

Comment (TomasTomecek) [reply content/posts/fedora-eln.md:49]: since we can't agree, I'll rewrite it completely

Comment (csomh) [reply content/posts/fedora-eln.md:49]: The part that does not sound right to me is "pushes to a branch".

**Review by csomh — APPROVED** [review]
:+1:
